object.js

```diff
@@ -289,6 +428,18 @@ function hint(gj, options) {
         MultiPolygon: MultiPolygon
     };
 
+    function propertiesLowerCase(obj) {
+        var props = [];
```

Comment:
for lookups like this, it'd be more efficient to use `.reduce` to create an object with `.linestring = true` rather than using `.indexOf`. indexOf is o(n), property access is constant for small objects